README.md

```diff
@@ -1,2 +1,30 @@
-# devworkspace-api
-K8S Api for a cloud develoment workspace specification [Draft proposal]
+# DevWorkspace API
+
+K8S-native Api for a cloud develoment workspace specification [Draft proposal]
```

Comment:
This sentence already in the repo description. I think it's redundant here. And I would rephrase it "Kube-native API for cloud development workspaces specification".